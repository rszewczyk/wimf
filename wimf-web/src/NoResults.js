// @flow
import React from "react";
import { css } from "glamor";

const noResultsStyle = css({
  display: "flex",
  alignItems: "center",
  justifyContent: "center",
  height: "280px",
  margin: "1rem 2rem 1rem 1rem",
  backgroundColor: "#F0F0F0",
  "> div": {
    margin: "1rem"
  }
});

export default function NoResults() {
  return (
    <div {...noResultsStyle}>
      <div>
        No Results. Do you have a filter applied? Try widening your criteria.
      </div>
    </div>
  );
}
