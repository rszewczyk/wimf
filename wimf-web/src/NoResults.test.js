import React from "react";
import renderer from "react-test-renderer";
import NoResults from "./NoResults";

it("renders correctly", () => {
  expect(renderer.create(<NoResults />).toJSON()).toMatchSnapshot();
});
