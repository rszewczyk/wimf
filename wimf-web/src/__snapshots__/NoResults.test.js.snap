// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  data-css-pxuxko=""
>
  <div>
    No Results. Do you have a filter applied? Try widening your criteria.
  </div>
</div>
`;
